scalecodec/type_registry/polymesh.json

```diff
@@ -715,7 +731,7 @@
     "PipId": "u32",
     "ProposalState": {
       "type": "enum",
-      "value_list": ["Pending", "Cancelled", "Rejected", "Scheduled", "Failed", "Executed", "Expired"]
+      "value_list": ["Pending", "Rejected", "Scheduled", "Failed", "Executed", "Expired"]
```

Comment:
@VictorVicente 
I am currently working on proposals [NCBD-97](https://polymath.atlassian.net/browse/NCBD-97)
and the requirement shows the following statuses: closed, open, deleted, rejected, scheduled, enacted
why are those different here?